ts/components/wallet/TransactionsList.tsx

```diff
@@ -84,7 +84,7 @@ export default class TransactionsList extends React.Component<Props> {
       <Row>
         <Left>
           <Text>
-            {isNew && <IconFont name="io-new" style={styles.newIconStyle} />}
```

Comment:
you no need anymore styles.newIconStyle
Could be removed from styles?

Reply:
Yes :)